### PR TITLE
lambda: getPhi for LSU ADV

### PR DESCRIPTION
**Pull request by rusefillc**
(cherry picked from commit dd3e3bf0d30c388ae3f19f3066cc91632b28f5d5)

### CODE REVIEW

File: firmware/lambda_conversion.cpp [hunk 1]
```diff
@@ -44,10 +44,26 @@ static float GetPhiLsu42(float pumpCurrent)
...
+    // Minimum lambda is 0.65
+    if (pumpCurrent < -1.38)
+    {
+        return 1 / 0.65f;
+    }
+
+    // Phi = -0.39025 * x + 0.00351 * x^2 + 0.99147, x < 0
+    // Phi = -0.98561 * x + 0.29302 * x^2 + 1.00547, x > 0
+    if (pumpCurrent < 0)
+        return (-0.39025 * pumpCurrent + 0.00351 * pumpCurrent * pumpCurrent + 0.99147);
```
Comment (mck1117) [inline firmware/lambda_conversion.cpp:64]: why not linear like the others? 0.003 quadratic term suggests that linear is a very good approximation, like it is for the others.

**Comment by mck1117** [comment]
substantially more accurate numbers used in e7bc7132d94863dcc31c1ef2f6050dbe4108e4ae (and yes, it does need the quadratic term for both rich and lean)

image

**Comment by rusefillc** [comment]
@mck1117 just to confirm https://github.com/mck1117/wideband/commit/e7bc7132d94863dcc31c1ef2f6050dbe4108e4ae is it the right one to mention here?

**Comment by mck1117** [comment]
off by one error, fixed

**Pull request closed without merging**